snapcraft/tests/test_lifecycle.py

```diff
@@ -724,6 +742,7 @@ def test_prime_with_build_info_records_manifest(self):
     prime: []
     stage: []
     stage-packages: []
+    uname: test uname
```

Comment:
Out of context this looks like a scriptlet - please use something that resembles the output of uname to avoid confusion.

Reply:
uname -a outputs something like this:
Linux juju-44eb60-remote-devel-0 4.10.0-32-generic #36~16.04.1-Ubuntu SMP Wed Aug 9 09:19:02 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

I would need to put the same string on every test here. So, shorter is better. And in many tests, we have "test-x" as the fake value for x.

Reply:
I said "something that resembles" for that reason.

My point was, `test uname` looks like a shell command. "test-x" doesn't look like a command.

Reply:
ok, I've replaced it with Linux test uname 4.10 x86_64.